CMakeLists.txt

```diff
@@ -170,6 +171,22 @@ if (LOMSE_BUILD_TESTS)
         link_directories( ${UNITTEST++_LIBRARY} )
         #message("UnitTest++ found: library: ${UNITTEST++_LIBRARY}" )
         #message("UnitTest++ found: include: ${UNITTEST++_INCLUDE_DIR}" )
+        # Check if UnitTest++'s class "Test" has a member named "next",
+        # that would mean UnitTest++ v1.4 or older
+        message(STATUS "Detecting UnitTest++ version")
+        file(WRITE "${CMAKE_CURRENT_BINARY_DIR}/unittest++-version-test.cpp"
+            "#include <UnitTest++.h>\n"
+            "int main() { UnitTest::Test* test; test->next = 0; return 0; }")
+        try_compile(LOMSE_UNITTEST_HAS_NEXT ${CMAKE_CURRENT_BINARY_DIR} "${CMAKE_CURRENT_BINARY_DIR}/unittest++-version-test.cpp"
+            CMAKE_FLAGS -DINCLUDE_DIRECTORIES=${UNITTEST++_INCLUDE_DIR})
+        file(REMOVE "${CMAKE_CURRENT_BINARY_DIR}/unittest++-version-test.cpp")
+        if(LOMSE_UNITTEST_HAS_NEXT)
+            message(STATUS "Detecting UnitTest++ version - done (<1.5)")
+            set(LOMSE_UNITTEST_OLD "1")
+        else()
+            message(STATUS "Detecting UnitTest++ version - done (>=1.5)")
+            set(LOMSE_UNITTEST_OLD "1")
```

Comment:
Shouldn't be  `set(LOMSE_UNITTEST_OLD "0")` ??

Reply:
Of course! Forgot to set it back to "0" after doing simulation tests in a final clean PR.

Sorry for that.

Submitted the fix.